snapshot: add CLI documentation (#466)
meilisearch/documentation: guides/advanced_guides/configuration.md

[To know more about storage in MeiliSearch look at this guide](/resources/about_storage.md)

**Default value**: `107374182400` (100 GiB)

### Schedule Snapshot Creation

**CLI option**: `--snapshot-path`

> The directory path where meilisearch will create snapshot,

Reviewer: ```suggestion
The directory path where MeiliSearch will create snapshots.
```

> If this command is not provided, snapshoting is deactivated.

Reviewer 2: ```suggestion
If this command is not called, snapshoting is deactivated.
```

### Snapshot interval

**CLI option**: `--snapshot-interval-sec`

> defines time gap (seconds) between each snapshot creation.

Reviewer: ```suggestion
Defines the time gap (seconds) between each snapshot creation.
```

Reviewer 2: What about?

```suggestion
Defines the time gap in seconds between each snapshot creation.
```

> Require `--snapshot-path` to be defined.

Reviewer 2: ```suggestion
Requires `--snapshot-path` to be defined.
```

**Default value**: `86400` (1 day)

### Init from snapshot

**CLI option**: `--init-from-snapshot`

> The snapshot file path to import.

Reviewer 2: ```suggestion
The path of the snapshot file to import.
```

> This command stop the process if:

Reviewer 2: ```suggestion
This command will stop the process if:
```

> - a database already exist

Reviewer 2: ```suggestion
- a database already exists
```

> - no snapshot exist at provided path.

Reviewer 2: ```suggestion
- No snapshot exists in the given path.
```

> If this command is not provided, no snapshot will be imported.

Reviewer 2: ```suggestion
If this command is not called, no snapshot will be imported.
```

Uncompatible with `--recover-from-snapshot`

### Recover from snapshot

**CLI option**: `--recover-from-snapshot`

> The snapshot file path to import.

Reviewer 2: ```suggestion
The path of the snapshot file to import.
```

> This command skip importation and continue the process if:

Reviewer 2: ```suggestion
This command will skip importing a snapshot and continue the process if:
```

> - a database already exist

Reviewer 2: ```suggestion
- A database already exists
```

> - no snapshot exist at provided path.

Reviewer 2: ```suggestion
- No snapshot exists in the given path.
```

> If this command is not provided, no snapshot will be imported.

Reviewer 2: ```suggestion
If this command is not called, no snapshot will be imported.
```
